denote-to-logseq.el

```diff
@@ -0,0 +1,63 @@
+;;; package -- Utilities to migrate denote notes to logseq
+;;
+;; Author: hapst3r
+;; Version: 0.1
+;; URL: https://github.com/bitspook/notes-migrator/denote-to-logseq.el
+;; Keywords: Denote, Logseq
+;; Version: 0.1.0
+;; Package-Requires: ((emacs "27.2") (denote "1.0.0")
+;;
+;;; Commentary:
+;; A WIP package for migrating Denote notes to Logseq
```

Comment:
Is this still a work-in-progress or are you ready to merge it into master (main) branch? If you're ready, please change this comment to remove the WIP mention.

Reply:
I'll be honest, I'd prefer if you decided this. I honestly don't know which criteria it would have to fulfill to be ready.

However, given that migration now respects denote silos, it features everything I'd want it to have - at least for the moment.

Reply:
> it features everything I'd want it to have - at least for the moment.

That is the criteria that matters imo. 

That's how I usually approach the open-source software I build. Write for yourself, present it well enough that someone with *exact same* use-case can use it too. If someone wants more from the software, they can communicate (e.g create a Github issue); and then you can decide if you have time and energy to extend it and in which manner. Otherwise they can always fork it and make their own changes.